lms/static/js/instructor_dashboard/membership.js

```diff
@@ -380,7 +380,7 @@ such that the value can be defined later than this assignment (file load order).
             this.$container = $container;
             this.$identifier_input = this.$container.find("textarea[name='student-ids-for-beta']");
             this.$btn_beta_testers = this.$container.find("input[name='beta-testers']");
-            this.$checkbox_autoenroll = this.$container.find("input[name='auto-enroll']");
+            this.$checkbox_autoenroll = this.$container.find("input[name='auto-enroll-beta']");
```

Comment:
Is this referring to auto-enrollment in the "Batch Enrollment" dialog? That is handled separately: 

https://github.com/edx/edx-platform/blob/1a49f4e230149c3f1f1994501fb9ff5e00ee0d87/lms/static/js/instructor_dashboard/membership.js#L539